examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -60,11 +60,12 @@
 
 # Define the dimension reduction to be used.
 # Here we use a classical univariate feature selection based on F-test,
-# namely Anova. We set the number of features to be selected to 500
-from sklearn.feature_selection import SelectKBest, f_classif
-feature_selection = SelectKBest(f_classif, k=500)
+# namely Anova. When doing full-brain analysis, it is better to use SelectPercentile, 
```

Comment:
Minor comment:
Can you please fix flake8 errors ?

Characters need not be too long more than 79 and there is a white space introduced after SelectPercentile,' '.
You can have have a look at this link for full details. From L344
https://travis-ci.org/nilearn/nilearn/jobs/204979982#L344